### PR TITLE
Client class improvements

### PR DESCRIPTION
**Pull request by chaudum**

### CODE REVIEW

File: src/Crate/PDO/Http/Client.php [hunk 1]
```diff
@@ -172,6 +172,18 @@ public function setHttpHeader($name, $value)
     }
 
     /**
+     * Set the default schema for PDO connection
+     *
+     * @param string $schemaName
+     *
+     * @return void
+     */
+    public function setDefaultSchema($schemaName)
```
Comment (Ocramius) [inline src/Crate/PDO/Http/Client.php]: Should be a constructor argument, no?

Comment (Ocramius) [inline src/Crate/PDO/Http/Client.php]: Nvm, I see that also the rest of the API uses setters :-\

**Comment by mikethebeer** [comment]
ack

**Comment by mikethebeer** [comment]
ack

**Pull request merged**